### PR TITLE
Update jeopardy.js

### PR DESCRIPTION
**Pull request by JavaIsTheWorst**
Jeopardy: Allow for the changing of the length of the answering window after player buzzes in and the length of the answering window for finals with /jp timer [seconds] and /jp finaltimer [seconds] (MIT licensed)

### CODE REVIEW

**Comment by Zarel** [comment]
Someone want to code review?

File: chat-plugins/jeopardy.js [hunk 1]
```diff
@@ -731,6 +733,26 @@ exports.commands = {
 			if (user.userid !== room.game.host.userid) return this.errorReply("This command can only be used by the host.");
 			room.game.nextQuestion();
 		},
+
+		timer: function (target, room, user) {
+			if (!room.game || room.game.gameid !== 'jeopardy') return this.errorReply("There is no game of Jeopardy going on in this room.");
+			if (user.userid !== room.game.host.userid) return this.errorReplay("This command can only be used by the host.");
+			let amount = parseInt(target);
+			if (!amount || amount < 2 || amount > 120) return this.errorReply("The amount must be a number between 5 and 300.");
```
Comment (CheeseMuffin) [inline chat-plugins/jeopardy.js]: The amounts don't seem to line up with the error message

File: chat-plugins/jeopardy.js [hunk 2]
```diff
@@ -731,6 +733,26 @@ exports.commands = {
...
+
+			room.game.answeringTime = amount;
+			this.addModCommand(`${user.name} has set the answering window for questions to ${amount} seconds`);
+		},
+
+		finaltimer: function (target, room, user) {
+			if (!room.game || room.game.gameid !== 'jeopardy') return this.errorReply("There is no game of Jeopardy going on in this room.");
+			if (user.userid !== room.game.host.userid) return this.errorReplay("This command can only be used by the host.");
+			let amount = parseInt(target);
+			if (!amount || amount < 2 || amount > 300) return this.errorReply("The amount must be a number between 5 and 300.");
```
Comment (CheeseMuffin) [inline chat-plugins/jeopardy.js]: Same here

File: chat-plugins/jeopardy.js [hunk 3]
```diff
@@ -731,6 +733,26 @@ exports.commands = {
...
+			let amount = parseInt(target);
+			if (!amount || amount < 2 || amount > 120) return this.errorReply("The amount must be a number between 5 and 300.");
+
+			room.game.answeringTime = amount;
+			this.addModCommand(`${user.name} has set the answering window for questions to ${amount} seconds`);
+		},
+
+		finaltimer: function (target, room, user) {
+			if (!room.game || room.game.gameid !== 'jeopardy') return this.errorReply("There is no game of Jeopardy going on in this room.");
+			if (user.userid !== room.game.host.userid) return this.errorReplay("This command can only be used by the host.");
```
Comment (CheeseMuffin) [inline chat-plugins/jeopardy.js]: errorReplay instead of errorReply

File: chat-plugins/jeopardy.js [hunk 4]
```diff
@@ -731,6 +733,26 @@ exports.commands = {
 			if (user.userid !== room.game.host.userid) return this.errorReply("This command can only be used by the host.");
 			room.game.nextQuestion();
 		},
+
+		timer: function (target, room, user) {
+			if (!room.game || room.game.gameid !== 'jeopardy') return this.errorReply("There is no game of Jeopardy going on in this room.");
+			if (user.userid !== room.game.host.userid) return this.errorReplay("This command can only be used by the host.");
```
Comment (CheeseMuffin) [inline chat-plugins/jeopardy.js]: error Replay instead of errorReply

File: chat-plugins/jeopardy.js (hunk 2 above)
Comment (CheeseMuffin) [inline chat-plugins/jeopardy.js]: Sorry, same as in the amounts don't match up

**Comment by JavaIsTheWorst** [comment]
Fixed, CheeseMuffin :)

**Comment by Zarel** [comment]
Looks simple enough to me, I guess.

**Pull request merged**